packages/web3torrent/src/utils/useInterval.ts

```diff
@@ -1,6 +1,6 @@
 import {MutableRefObject, useEffect, useRef} from 'react';
 
-export function useInterval(callback, delay = 0) {
+export function useInterval(callback, delay: number | boolean = 0) {
```

Comment:
Why `number | boolean`?

Reply:
'cause it's the way to stop the setInterval. I thought that that could be a good way to stop/pause the interval. If I pass false, the setInterval pauses.